WebApp/webapp/src/pages/Prediction/index.test.tsx

```diff
@@ -0,0 +1,90 @@
+import { render, screen } from '@testing-library/react'
```

Comment:
Tests need to match with the page. Instead of <matchups> it should be the prediction page, and test the components inside that page